### ISSUE TITLE
Error when making rhythm implicit per line

### ISSUE THREAD

**Issue by dc7kr** [frescobaldi/frescobaldi#1337]
When making the rhythm implicit per line, Frescobaldi removes the duration of notes in pickup bars which causes broken lilypond code (tested again in 3.1.2, seen in older versions too)

Steps to reproduce: 

1. Create the following lilypond file:

```
\version "2.18.0"

violinOneMusic =  \relative c' {
  \time 4/4
  \key c \major
  \partial 8 r8
  
  c4 d4 e4 f4
}
```
2. Mark the entire code and select "Tools->Musical Transformations->Rhythm->Make implicit (per line)" 

Result:

```
\version "2.18.0"

violinOneMusic =  \relative c' {
  \time 4/4
  \key c \major
  \partial 8 r
  
  c4 d e f
}
```

Expected: 

```
[...]
  \partial 8 r8
[...]
```

**Comment by fedelibre** [comment]
I confirm the bug.

Good to have it here, even though the fix should be made [in python-ly repository](https://github.com/frescobaldi/python-ly/blob/master/ly/rhythm.py#L322).

**Comment by wbsoft** [comment]
fixed in python-ly-0.9.7
